src/components/AccountTokens.tsx

```diff
@@ -0,0 +1,91 @@
+import { Button, Collapse, Stack, useDisclosure } from '@chakra-ui/react'
+import { CAIP19 } from '@shapeshiftoss/caip'
+import { useMemo } from 'react'
+import { FaArrowCircleDown } from 'react-icons/fa'
+import { useTranslate } from 'react-polyglot'
+import { generatePath } from 'react-router'
+import { Text } from 'components/Text'
+import { selectAssetByCAIP19 } from 'state/slices/assetsSlice/assetsSlice'
+import {
+  AccountSpecifier,
+  selectPortfolioAccountById
+} from 'state/slices/portfolioSlice/portfolioSlice'
+import { useAppSelector } from 'state/store'
+
+import { AccountRow } from './AccountRow/AccountRow'
+import { Card } from './Card/Card'
+
+export const AccountTokens = ({
+  caip19,
```

Comment:
let's continue this pattern of calling this `assetId` across the board, i should never have propagated this pattern of calling it `caip19` in the first place, it's kind of an implementation detail